### DISCUSSION TITLE
docs/guide/composition/instance/config

### DISCUSSION THREAD

**Discussion by giscus** [iPanelDev/docs#44] [Giscus]
# docs/guide/composition/instance/config

webSocket

https://ipaneldev.github.io/docs/guide/composition/instance/config

**Comment by chenchen71956** [comment]
在mcdr插件内，ipanel_mcdreforged/config.py中的self.custom_name = config.get('custonName')此处有误，应该为config.get('customName')

**Reply by chenchen71956** [reply]
这个错误会引发"config.customName"为空，将使用默认名称"MCDReforged#2.3.0.0"，即便我在配置文件中设置了名称

**Reply by Zaitonn** [reply]
感谢 会在下个版本修复（

如果急的话可以自行clone修改打包使用

**Reply by Zaitonn** [reply]
因为一些项目安排上的原因 最近可能不会马上更新iPanel :(

**Reply by chenchen71956** [reply]
没有关系，我会继续在我的服务器上使用iPanel，如果有新的问题会继续向您请教
